style.css

```diff
@@ -0,0 +1,217 @@
+body {
+    height: 842px;
+    width: 595px;
```

Comment:
weren't you supposed to use % instead of px?

Reply:
Done